README - react.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
So Create React App is unmaintained and probably not the greatest way to set up a new project today since it pulls in old dependencies with no way to update :/
I haven't used it yet but Vite seems like a project that's similar in spirit but actually maintained.